### PR TITLE
eslintrc cleanup and format code

### PR DESCRIPTION
**Pull request by RevinKey**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -35,6 +36,8 @@
     ]
   },
   "devDependencies": {
+    "eslint-config-airbnb": "^17.1.0",
```
Comment (ldayananda) [inline package.json]: according to this we need to add the peerDeps too https://github.com/airbnb/javascript/tree/master/packages/eslint-config-airbnb#eslint-config-airbnb-1

File: src/actions.js [hunk 2]
```diff
@@ -10,70 +10,66 @@ export const RECEIVE_VIDEO_ANALYTICS = 'RECEIVE_VIDEO_ANALYTICS';
...
+}
 
 export function updateSelectedVideo(video) {
-    return {
-        type: RECEIVE_SELECTED_VIDEO,
-        selectedVideo: video
-    }
+  return {
+    type: RECEIVE_SELECTED_VIDEO,
+    selectedVideo: video,
```
Comment (ldayananda) [inline src/actions.js]: extra comma? in a few of the below too

File: src/components/BrightcovePlayer.jsx [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+  render() {
+  	return (
+    <Player
+      attrs={{ id: 'videoPlayer' }}
+      accountId="6027103981001"
+      playerId="default"
+      onSuccess={this.success}
+      options={{
+  				controls: true,
```
Comment (ldayananda) [inline src/components/BrightcovePlayer.jsx]: this seems off 😅

File: src/configureStore.js [hunk 4]
```diff
@@ -1,11 +1,11 @@
...
 export default function configureStore() {
-    return createStore(
-        rootReducer,
-        // lets us dispatch functions
-        applyMiddleware(thunkMiddleware)
-    );
+  return createStore(
+    rootReducer,
+    // lets us dispatch functions
+    applyMiddleware(thunkMiddleware),
```
Comment (ldayananda) [inline src/configureStore.js:9]: extra comma again

File: src/serviceWorker.js [hunk 5]
```diff
@@ -11,125 +11,125 @@
 // opt-in, read https://bit.ly/CRA-PWA
 
 const isLocalhost = Boolean(
-    window.location.hostname === 'localhost' ||
+  window.location.hostname === 'localhost'
```
Comment (ldayananda) [inline src/serviceWorker.js]: this looks wrong

File: package.json [hunk 6]
```diff
@@ -17,7 +17,8 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "lint": "eslint . --fix"
```
Comment (ldayananda) [inline package.json:21]: based on how many weird changes there were, maybe we shouldn't auto fix

File: src/serviceWorker.js (hunk 5 above)
Comment (RevinKey) [reply src/serviceWorker.js]: I think it breaks it up because of line length and the comments

Comment (ldayananda) [reply src/serviceWorker.js]: I think the `||` got removed though because the next line is a comment

**Review by ldayananda — APPROVED** [review]

**Pull request merged**